frontend/src/pages/GamePage/MainScene.tsx

```diff
@@ -1,8 +1,9 @@
 import Phaser from 'phaser';
-import { GameData } from 'types/game';
+import { GameData, GameRoomInfoType } from 'types/game';
 
 import { sockets } from 'hooks';
 import { NavigateFunction } from 'react-router-dom';
+import { MatchHistoryType } from 'types/profile';
```

Comment:
```suggestion
import { NavigateFunction } from 'react-router-dom';

import { GameData, GameRoomInfoType } from 'types/game';
import { MatchHistoryType } from 'types/profile';
import { sockets } from 'hooks';
```

<img width="367" alt="image" src="https://user-images.githubusercontent.com/37893979/221159566-1264f3d1-b9ae-4513-98d2-2580391d4475.png">